bin/crunchy-postgres-exporter/start.sh

```diff
@@ -214,7 +214,7 @@ else
     fi
 fi
 
-sed -i "s/#PGBACKREST_INFO_THROTTLE_MINUTES#/${PGBACKREST_INFO_THROTTLE_MINUTES:-10}/g" /tmp/queries.yml
+sed -i -e "s/#PGBACKREST_INFO_THROTTLE_MINUTES#/${PGBACKREST_INFO_THROTTLE_MINUTES:-10}/g" -e "s/#PG_STAT_STATEMENTS_LIMIT#/${PG_STAT_STATEMENTS_LIMIT:-100}/g" /tmp/queries.yml
```

Comment:
```suggestion
sed -i -e "s/#PGBACKREST_INFO_THROTTLE_MINUTES#/${PGBACKREST_INFO_THROTTLE_MINUTES:-10}/g" -e "s/#PG_STAT_STATEMENTS_LIMIT#/${PG_STAT_STATEMENTS_LIMIT:-20}/g" /tmp/queries.yml
```

Modeling the upstream pgMonitor change for `PG_STAT_STATEMENTS_LIMIT`:

https://github.com/CrunchyData/pgmonitor/blob/41ef7002498c9635880b0054642fd9dd300cc66c/exporter/postgres/sysconfig.postgres_exporter_pg13#L9